scripts/init_db.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS customers (
+    id VARCHAR(26) PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS address (
+    id VARCHAR(26) PRIMARY KEY,
+    customer_id VARCHAR(26) NOT NULL,
+    street VARCHAR(255) NOT NULL,
+    city VARCHAR(100) NOT NULL,
+    state VARCHAR(50) NOT NULL,
+    postal_code VARCHAR(20) NOT NULL,
+    country VARCHAR(100) NOT NULL,
+    FOREIGN KEY (customer_id) REFERENCES customers(id) ON DELETE CASCADE
+);
```

Comment:
here too, I believe that this data structure is initial and that we will soon change. Maybe it makes sense to put versions for these .sql

Reply:
this delete on cascade is used to delete the address automatically after the customer delete